### PR TITLE
CBRS Small Cell Model swapped for commerical names

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

File: docs/5g-on-helium/5g-on-helium.mdx [hunk 1]
```diff
@@ -11,72 +11,34 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 
 # 5G On Helium
 
-Today all mobile networks are built top-down by service providers (e.g., AT&T and Verizon in the
+Today all mobile networks are built top-down by service providers (e.g., AT&T and Verizon in The
```
Comment (abhay) [inline docs/5g-on-helium/5g-on-helium.mdx]:
```suggestion
Today all mobile networks are built top-down by service providers (e.g., AT&T and Verizon in the
```

Comment (abhay) [reply docs/5g-on-helium/5g-on-helium.mdx]: doesn't need to be capitalized.

File: docs/5g-on-helium/cbrs-radios.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+channel is most relevant for using 5G on the Network.
+
+## Outdoor Small Cell Radios
+
+Outdoor Small Cell Radios are more powerful than Indoor Small Cell Radios and are weatherized to
+endure the elements. Outdoor Small Cell Radios create a larger area of 5G coverage while requiring a
+bit more planning when installing and maintaining.
+
+Outdoor Small Cell Radios require installation above ground level and require registration with the
+Spectrum Access System (SAS) to operate. Please be sure to evaluate your locations and ability to
```
Comment (cokes518) [inline docs/5g-on-helium/cbrs-radios.mdx]:
```suggestion
Spectrum Access System (SAS) to operate. Please be sure to evaluate your site location and ability to
```

File: docs/5g-on-helium/cbrs-radios.mdx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+install these Small Cell Radios, and consider hiring a professional to ensure your installation is
+secure and meets all regulatory requirements.
+
+SAS registration is expected to be completed within 5 days of submission, Hotspot Owners will be
```
Comment (cokes518) [inline docs/5g-on-helium/cbrs-radios.mdx]:
```suggestion
SAS registration is expected to be completed within 5 days of submission. Hotspot Owners will be
```

File: docs/5g-on-helium/cbrs-radios.mdx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+explanation of what needs to be corrected.
+
+## Indoor Small Cell Radios
+
+Indoor Small Cell Radios are less powerful and are ideal for installations such as coffee shops,
+bars, grocery stores, etc. Indoor Small Cell Radios are generally easier to install and create a
+smaller, localized area of 5G coverage.
+
+Indoor Small Cell Radios are physically smaller radios and can be installed at any height.
+Installing an Indoor Small Cell Radio outdoors is not recommended as they are not weather resistant.
```
Comment (cokes518) [inline docs/5g-on-helium/cbrs-radios.mdx]:
```suggestion
**Installing an Indoor Small Cell Radio outdoors is not recommended as they are not weather resistant.**
```

File: docs/5g-on-helium/cbrs-radios.mdx [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+past 10 minutes. Small Cell Location will be either `Indoor` or `Outdoor` depending on the Small
+Cell Radio type.
+
+:::note 5G Hotspots with more than one Small Cell Radio
+
+Small Cell Status will reflect all attached Small Cell Radios in aggregate, if any connected Small
+Cell Radio has broadcast a `heartbeat` to the SAS, the Status will be `Active` regardless of the
+actual status of any other connected Small Cell Radio.
+
+Refer to the 5G Hotspot dashboard for information on each connected Small Cell Radio
```
Comment (cokes518) [inline docs/5g-on-helium/cbrs-radios.mdx]:
```suggestion
Refer to the 5G Hotspot dashboard for information on each connected Small Cell Radio (local network connection only)
```

File: docs/5g-on-helium/cbrs-radios.mdx [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+### Small Cell Radios Need GPS
+
+All compatible CBRS Small Cell Radios use GPS for network timing data and to assert the Small Cell
+Radio's location when requesting approval to transmit from the FCC-regulated Spectrum Access System
+(SAS).
+
+Small Cell Radios will need to be placed somewhere that allows the GPS antenna to be in a location
+with good GPS reception (i.e. near a window). Indoor Small Cell Radios are shipped with an external
+GPS antenna that can be positioned indipendently while Outdoor Small Cell Radios have the GPS antena
```
Comment (cokes518) [inline docs/5g-on-helium/cbrs-radios.mdx]:
```suggestion
GPS antenna that can be positioned independently while Outdoor Small Cell Radios have the GPS antenna
```

**Review by cokes518 — APPROVED** [review]
To Do: better explanation on DPA and inline map view of priority coverage

**Pull request merged**